modules/howtos/examples/request-tracer.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
```

Comment:
Just wondering are we using this example anywhere? Or is this prep for referencing later?

Also this will need a new test.

Reply:
hmm yes, looks like this needs some more work, ta